installation/vs-code-integration/new-project-wizard.md

```diff
@@ -40,7 +40,7 @@ You can start the Telerik UI for .NET MAUI Template Wizard in either of the foll
 
     1. Select **Telerik UI for .NET MAUI Template Wizard: Launch** from the menu and press **Enter**. 
     1. The **Telerik UI for .NET MAUI Template Wizard** opens
-    1. Using the Template Wizard you can create a new **Blank App** or **Blank Shell App**.
+    1. Using the Template Wizard you can create a new **Blank App** or **Blank Shell App**. You can create a project with a predefined [theme]({%slug themes-overview%}) using the **USE TELERIK THEMING** option.
 
     ![Telerik UI for .NET MAUI VS Code snippets](images/MauiTemplateWizard.gif)
 
```

Comment:
I noticed that there is a section that doesn't belong in this article. See [line 47](https://github.com/telerik/maui-docs/blob/270b0b78cb0109b7428716c673eca848ef6b322c/installation/vs-code-integration/new-project-wizard.md?plain=1#L47). You may want the `## Set Up the Telerik NuGet Server` from this article. I don't see this option in the wizard.